systemd/ceph-osd@.service

```diff
@@ -19,6 +19,7 @@ TasksMax=infinity
 Restart=on-failure
 StartLimitInterval=30min
 StartLimitBurst=3
+RestartSec=5min
```

Comment:
My concern is when the osd service exit accidently, such as: killed by someone. If we wait 5mins before restart, it will be too long